acceptancetests/assess_recovery.py

```diff
@@ -64,6 +64,15 @@ def enable_ha(bs_manager, controller_client):
         controller_client, bs_manager.known_hosts)
     bs_manager.known_hosts = remote_machines
 
+def disable_ha(bs_manager, controller_client):
+    """Disable HA and wait for the controllers to be ready."""
+    log.info("Disabling HA.")
+    controller_client.wait_for(controller_client.remove_machines(["1", "2"], controller=True))
+    show_controller(controller_client)
+    remote_machines = get_remote_machines(
+        controller_client, None)
+    bs_manager.known_hosts = remote_machines
```

Comment:
I think it'd be better for this to just return the remote_machines vs monkey with a bs_manager that's not directly tied to this.

Reply:
So this is how enable_ha does it, so just following suit.